test/cloud_integration_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	. "github.com/onsi/gomega"
 )
 
-var _ = Describe("Cloud Mode Integration Tests", func() {
+var _ = Describe(versionedPrefixForQuarantining()+"Cloud Mode Integration Tests", func() {
```

Comment:
@kylekthompson where do I find the identity for go tests? Will prefixing the top level description with a version number be enough to quarantine version-specific test deprecations?